### DISCUSSION TITLE
Synchronize with Nextcloud maps

### DISCUSSION THREAD

**Discussion by keunes** [organicmaps/organicmaps#776] [Ideas]
I use the [Nextcloud Maps](https://github.com/nextcloud/maps) app for holiday planning. As both an Organic Maps and Nextcloud Maps user, I would love to have synchronization of my pins between the two.

Official synchronization is all the more important because of changes in Android 11 I as a user can no longer access the kml files on my phone. With that, the option to use the Android Nextcloud app to synchronise these files is gone.

The Lists in Organic Maps would be the Categories as they are in Nextcloud Maps.

It seems that there is no API yet for Nextcloud Maps, but there is a [request over at Nextcloud Maps to allow for synchronisation](https://github.com/nextcloud/maps/issues/92) with other apps. There I've added a request to sync with Organic Maps as well.

**Comment by penguin86** [comment]
Hello @keunes!
[There is now finally an API for Nextcloud Maps](https://github.com/organicmaps/organicmaps/issues/622#issuecomment-896712451)!
I used it to develop **a very basic app that allows to open the Nextcloud Maps geobookmarks in any android app** supporting Geo URI (not only Organic Maps, but many others, including Google Maps...). Don't expect bells and whistles, it's the bare minimum to get me on the road again, but I hope it can help you until a proper syncronization is implemented in Organic Maps...

![Screenshot](https://raw.githubusercontent.com/penguin86/nextcloud-maps-client/master/screenshots/2.png)

You can download from [the github page](https://github.com/penguin86/nextcloud-maps-client/releases) and soon will be available on [F-Droid](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9655).

Keep in mind that ATM only Geobookmarks are supported, so you can use it to plan your holidays, but will not be able to see your holidays photos on a map. I plan to add this feature in the future...

Have a nice day :)

**Reply by alejandrogallo** [reply]
your app fails to start in android 13

**Reply by penguin86** [reply]
Hello Alejandro,
it doesn't happen on my Android 13 phone, so it must depend on something else. I need your help to pinpoint the problem, but this is off topic here, [please file a bug here](https://github.com/penguin86/nextcloud-maps-client/issues).
PS: Please check to have v0.3.6, if you installed from Play Store you have an older version with a known bug on malformed datasets. If possible, [install the app through F-Droid](https://f-droid.org/it/packages/it.danieleverducci.nextcloudmaps/).
